libraries/AP_GPS/AP_GPS_ERB.cpp

```diff
@@ -159,8 +159,8 @@ AP_GPS_ERB::_parse_gps(void)
         state.location.alt    = (int32_t)(_buffer.pos.altitude_msl * 100);
         state.status          = next_fix;
         _new_position = true;
-        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f;
-        state.vertical_accuracy = _buffer.pos.vertical_accuracy * 1.0e-3f;
+        state.horizontal_accuracy = _buffer.pos.horizontal_accuracy * 1.0e-3f; // RMS
```

Comment:
I thought I had seen an e-mail about this, but now I can't find the comment anywhere, but I don't see anything in the ERB sheet that indicates characterizing this as RMS is correct. (And I admit I hesitated before peaking at RTKlib :) a quick google search filed to yield an anwser)